google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -211,17 +287,63 @@ public static TransportChannelProvider defaultTransportChannelProvider() {
    * <p>Please note that these settings will affect both manually batched calls
    * (bulkMutateRowsCallable) and automatic batched calls (bulkMutateRowsBatchingCallable). The
    * {@link RowMutation} request signature is ignored for the manual batched calls.
+   *
+   * <p>Retries will be attempted if RPC are failed with {@link Code#DEADLINE_EXCEEDED}, {@link
+   * Code#UNAVAILABLE} or {@link Code#ABORTED} failure code.
+   *
+   * <p>RetryDelay starts with 100ms or 0.1 seconds, which multiplies with 1.3 times on each
+   * reattempt till a max retry timeout value of 60 seconds.
+   *
+   * <p>RPC timeout is 20 seconds.
+   *
+   * <p>Total timeout for this operation is 600 seconds or 10 mins.
+   *
+   * <p>On breach of certain triggers, this batch will initiates processing of accumulated requests.
```

Comment:
...this batch will initiate processing of..or this batch initiates the processing of ...